protected/controllers/GpController.php

```diff
@@ -60,10 +65,10 @@ public function actionView($id)
      */
     public function actionCreate($context = null)
     {
+        Yii::app()->assetManager->RegisterScriptFile('js/Gp.js');
```

Comment:
I'm not a huge fan of having capital letters in a javascript filename, but there do seem to be a few other files that use this convention.